Unreal/SystemLocalized/det/UnrealI.det

```diff
@@ -68,13 +68,13 @@ MenuName="Gassack"
 
 [GiantGasbag]
 ; EN: NameArticle=" the "
-NameArticle=" der "
+NameArticle=" ein "
 ; EN: MenuName="Giant Gasbag"
 MenuName="Riesiger Gassack"
 
 [GiantManta]
 ; EN: NameArticle=" the "
-NameArticle=" der "
+NameArticle=" ein "
```

Comment:
Similar scenario to the Giant Gasbag, see comment in Line 71.

Reply:
@SlyMk3 See response in 71